extensions/userAgent/userAgent.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * Sets a user agent according to --user-agent or -phone or -tablet options
```

Comment:
```suggestion
 * Sets a user agent according to --user-agent or --phone or --tablet options
```

Missing `-` :slightly_smiling_face: